README.md

```diff
@@ -201,6 +201,14 @@ Then just start `PasswordlessLockActivity` from inside your `Activity`
 startActivity(lock.newIntent(this));
 ```
 
+#### Android App Links - Custom Scheme
+Currently the default scheme used for Web Authentication is `https`. This works best for Android API 23 or newer if you're using [Android App Links](https://developer.android.com/training/app-links/index.html), but in previous Android versions this may show the intent chooser dialog prompting the user to chose either your application or the browser to resolve the intent. You can change this behavior by using a custom unique scheme, so that the OS opens directly the link with your app.
```

Comment:
`Currently the default scheme used for Web Authentication` -> `Currently the default scheme used for the redirect url used to send the Web Auth results`